src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -773,31 +768,39 @@ func (c controller) waitForOLMOperators() bool {
 		return true
 	}
 	for _, operator := range operators {
-		csvName, err := c.kc.GetCSVFromSubscription(operator.Namespace, operator.SubscriptionName)
-		if err != nil {
-			c.log.WithError(err).Warnf("Failed to get subscription of operator %s", operator.Name)
-			continue
-		}
 
-		csv, err := c.kc.GetCSV(operator.Namespace, csvName)
-		if err != nil {
-			c.log.WithError(err).Warnf("Failed to get %s", operator.Name)
-			continue
-		}
+		//FIXME: We check the StorageCluster of the OCS operator WA bug 1968606
+		//Remove this code when bug 1968606 is fixed
+		sc, err := c.kc.GetStorageCluster(operator.Namespace)
+		if sc != nil {
```

Comment:
It is still very specific for ocs operator. What do you think if we could define what should we watch to get the status? In this case we could make this decision in operator specific code in the service and here we could use it to track CSV or any other specific CR.

Reply:
Yes, I think that could work